Add alert on silent notifications with title and body (#6438)
braze-inc/braze-docs: _docs/_developer_guide/platform_integration_guides/ios/push_notifications/silent_push_notifications.md

To send a silent push notification, set the `content-available` flag to `1` in a push notification payload. When sending a silent push notification, you might also want to include some data in the notification payload, so your application can reference the event. This could save you a few networking requests and increase the responsiveness of your app.

{% alert warning %}

> To ensure that the notification is truly silent, be sure to exclude the title and body when setting the `content-available` flag to `1.` While APNs does not strictly enforce this exclusivity, attaching both a title and body with `content-available=1` is not recommended, as it can lead to undefined behavior. For further details, refer to the official [Apple documentation on background updates](https://developer.apple.com/documentation/usernotifications/setting_up_a_remote_notification_server/pushing_background_updates_to_your_app).

Reviewer: ```suggestion
Attaching both a title and body with `content-available=1` is not recommended because it can lead to undefined behavior. To ensure that a notification is truly silent, exclude both the title and body when setting the `content-available` flag to `1.` For further details, refer to the official [Apple documentation on background updates](https://developer.apple.com/documentation/usernotifications/setting_up_a_remote_notification_server/pushing_background_updates_to_your_app).
```

{% endalert %}

> 

Reviewer 2: I double checked our wording on the Dashboard, and we do explicitly use the wording `Content-Available` when making the campaign (Push campaign > iOS only) so we should be good here!

<img width="924" alt="Screenshot 2023-10-30 at 5 31 01 PM" src="https://github.com/braze-inc/braze-docs/assets/4797040/69d77719-66aa-43bb-9dcf-d1f223132948">
